### PR TITLE
WIP Task Prime Numbers

### PR DESCRIPTION
**Pull request by trump-ny**

### CODE REVIEW

**Comment by trump-ny** [comment]
PLEASE SEE TASK PRIME NUMBERS!!!!     TASKS Array Butterfly, MinMaxValue in Array ,RandomGeneratorDemo DON'T LOOK....  Accidentally added in commit.

**Review by VasiliZas — APPROVED** [review]

File: homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                sum +=i;
+            }
+            sumWithout17and71 = sum - (17+71);
+        }
+        System.out.print(sumWithout17and71);
+
+    }
+
+
+
```
Comment (KaydunovDenis) [inline homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java:29]: delete empty lines

**Review by KaydunovDenis — APPROVED** [review]

**Review by PavelBoPavel — APPROVED** [review]

File: homework/yaroslav_zhimanov/array/butterfly/Butterfly.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                else
+                    arr[a][b] = 1;
+            }
+        }
+        for (a = 0; a < arr.length; a++) {
+            for (b = 0; b < arr[a].length; b++) {
+                System.out.print(arr[a][b]);
+                //Логгер не до конца понимаю. Разберусь, тогда буду писать логгер.
+            }
+            System.out.println(" ");
```
Comment (max-java) [inline homework/yaroslav_zhimanov/array/butterfly/Butterfly.java:34]: You'd better start adding logger right now. 
Do not hesitate to ask questions in lectures if you don't understand smth. 
It is very important to not to study alone or put off important materials for later

File: homework/yaroslav_zhimanov/array/min_max/MinMax.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        int array[] = new int[]{702, 547, 662, 932, 203, 312, 818, 29, 642, 925, 348, 256, 895, 851, 96, 450, 135, 533, 849, 736, 218, 350, 693, 462, 14, 181, 463, 81, 501, 94, 480, 662, 217, 550, 262, 561, 796, 79, 907, 607, 484, 720, 597, 337, 889, 255, 847, 598, 401, 323, 1400, 1713, 1959, 1681, 1842, 1090, 1269, 1400, 1331, 1490, 1837, 1108, 1804, 1407, 1804, 1475, 1641, 1949, 1749, 1670, 1138, 1282, 1994, 1084, 1661, 1579, 1685, 1069, 1119, 1066, 1035, 1096, 1725, 1990, 1657, 1390, 1597, 1983, 1224, 1235, 1949, 1801, 1445, 1924, 1658, 1717, 1308, 1108, 1826, 1387};
+
+
+        int max = getMax(array);
+        System.out.println("Максимальное число: " + max);
+
+
+        int min = getMin(array);
+        System.out.println("Минимальное число: " + min);
+
```
Comment (max-java) [inline homework/yaroslav_zhimanov/array/min_max/MinMax.java:15]: Move this into tests

File: homework/yaroslav_zhimanov/array/min_max/MinMax.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    public static int getMax(int[] inputArray) {
+        int maxValue = inputArray[0];
+        for (int i = 1; i < inputArray.length; i++) {
+            if (inputArray[i] > maxValue) {
+                maxValue = inputArray[i];
+            }
+
+
```
Comment (max-java) [inline homework/yaroslav_zhimanov/array/min_max/MinMax.java:25]: remove empty rows

File: homework/yaroslav_zhimanov/array/min_max/MinMax.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    public static int getMin(int[] inputArray) {
+        int minValue = inputArray[0];
+        for (int i = 1; i < inputArray.length; i++) {
+            if (inputArray[i] < minValue) {
+                minValue = inputArray[i];
+            }
+        }
+
+
```
Comment (max-java) [inline homework/yaroslav_zhimanov/array/min_max/MinMax.java:39]: remove empty rows

File: homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
+package for_homework;
```
Comment (max-java) [inline homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java:1]: you'd better not use underscore in package names. 
Use Sonar

File: homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+                if (i%j==0){
+                    break;
+                }
+            }
+            if (i==j){
+                sum +=i;
+            }
+            sumWithout17and71 = sum - (17+71);
+        }
+        System.out.print(sumWithout17and71);
```
Comment (max-java) [inline homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java:24]: encapsulate this into Class and Method

File: homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java [hunk 8]
```diff
@@ -0,0 +1,31 @@
+package for_homework;
+
+public class PrimeNumbers {
+    public static void main (String args []){
+
+        int n = 500;
+        int i;
+        int j;
+        int sum =0;
+        int sumWithout17and71=0;
```
Comment (max-java) [inline homework/yaroslav_zhimanov/for_homework/PrimeNumbers.java:10]: This looks like test case, move this to tests.

File: homework/yaroslav_zhimanov/random_generator/RandomGeneratorDemo.java [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+        Integer randoma = random.nextInt(1000);
+        Integer randomb = random.nextInt(1000);
+        Integer randomc = random.nextInt(1000);
+        Integer summa = randoma + randomb + randomc;
+
+        System.out.println("Случайное число 1: " + randoma);  //знаю про логгер. Дайте так хоть не запутаться)
+        System.out.println("Случайное число 2: " + randomb);
+        System.out.println("Случайное число 3: " + randomc);
+        System.out.println("Сумма всех сгенерированных чисел: " + summa);
```
Comment (max-java) [inline homework/yaroslav_zhimanov/random_generator/RandomGeneratorDemo.java:26]: Divide this into service and test cases

**Review by max-java — CHANGES_REQUESTED** [review]
Slit this into projects, separate for each one task.
Add quickstart project, and use sonar.

**Pull request closed without merging**